js/main.js

```diff
@@ -1,108 +1,123 @@
 var today = new Date();
 
 // Async HTTP GET Function
-var HttpClient = function() { // Thanks http://stackoverflow.com/a/22076667/1709894!
-  this.get = function(aUrl, aCallback) {
-    var anHttpRequest = new XMLHttpRequest();
-    anHttpRequest.onreadystatechange = function() {
-      if (anHttpRequest.readyState == 4 && anHttpRequest.status == 200)
-        aCallback(anHttpRequest.responseText);
+var HttpClient = function () { // Thanks http://stackoverflow.com/a/22076667/1709894!
```

Comment:
I know it's difficult to mess around with it, but could you tweak your indent settings? I really dislike messy diffs, and reindenting entire files is about as messy as it gets.